src/Windows.Core/DEVMODEW.cs

```diff
@@ -0,0 +1,783 @@
+        /// Specifies the frequency, in hertz (cycles per second), of the display device in a particular mode.
+        /// This value is also known as the display device's vertical refresh rate. Display drivers use this member.
+        /// It is used, for example, in the ChangeDisplaySettings function. Printer drivers do not use this member.
+        /// </summary>
+        /// <remarks>
+        /// When you call the <c>User32.EnumDisplaySettings</c> function, the <see cref="dmDisplayFrequency"/> member may return with the value 0 or 1.
+        /// These values represent the display hardware's default refresh rate.
+        /// This default rate is typically set by switches on a display card or computer motherboard, or by a configuration program that does not use display functions such as <c>User32.EnumDisplaySettings</c>.
+        /// </remarks>
+        /// <remarks>
+        /// This member is not used for printers.
+        /// </remarks>
+        [FieldOffset(120)]
+        public uint dmDisplayFrequency;
+
+        /// <summary>
+        /// Specifies one of the <see cref="DEVMODEW_DeviceIcmMethodOptions"/> constants.
+        /// </summary>
+        [FieldOffset(124)]
+        public DEVMODEW_DeviceIcmMethodOptions dmICMMethod;
+
+        /// <summary>
+        /// Specifies one of the <see cref="DEVMODEW_DeviceIcmIntentOptions"/> constants.
+        /// </summary>
+        [FieldOffset(128)]
+        public DEVMODEW_DeviceIcmIntentOptions dmICMIntent;
+
+        /// <summary>
+        /// Specifies one of the <see cref="DEVMODEW_DeviceMediaTypeOptions"/> constants.
+        /// </summary>
+        [FieldOffset(132)]
+        public DEVMODEW_DeviceMediaTypeOptions dmMediaType;
+
+        /// <summary>
+        /// Specifies one of the <see cref="DEVMODEW_DitherTypeOptions"/> constants.
+        /// </summary>
+        [FieldOffset(136)]
+        public DEVMODEW_DitherTypeOptions dmDitherType;
+
+        /// <summary>
+        /// Not used; must be zero.
+        /// </summary>
+        [FieldOffset(140)]
+        public uint dmReserved1;
+
+        /// <summary>
+        /// Not used; must be zero.
+        /// </summary>
+        [FieldOffset(144)]
+        public uint dmReserved2;
+
+        /// <summary>
+        /// This member must be zero.
+        /// </summary>
+        [FieldOffset(150)]
+        public uint dmPanningWidth;
+
+        /// <summary>
+        /// This member must be zero.
+        /// </summary>
+        [FieldOffset(154)]
+        public uint dmPanningHeight;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="DEVMODEW"/> struct
+        /// with the <see cref="dmSize" /> field initialized.
+        /// </summary>
+        /// <returns>
+        /// An instance of the structure.
+        /// </returns>
+        public static DEVMODEW Create() => new DEVMODEW { dmSize = (ushort)sizeof(DEVMODEW) };
+    }
+
+    public enum DEVMODEW_DisplayOptions : uint
+    {
```

Comment:
Although the enum values are missing from the docs, I found this in the header files:

```c

/* DEVMODE dmDisplayFlags flags */

// #define DM_GRAYSCALE            0x00000001 /* This flag is no longer valid */
#define DM_INTERLACED           0x00000002
#define DMDISPLAYFLAGS_TEXTMODE 0x00000004
```

They are *flags*, so can we rename this enum to be `DEVMODE_DISPLAYFLAGS` to match the struct name and better fit the member name that uses it?

Reply:
Yeah. Some of these comments I already converted to XML docs.
I'll also rename accordingly and decorate with `FlagsAttribute`.

BTW, do you want me to place all these enums each into a separate file?

Reply:
> do you want me to place all these enums each into a separate file?

My first inclination was yes, but given they are very much just to support one struct (and even named to signify that), I think one file is probably best.

Reply:
I've seen some occurrences in PInvoke where the flags are also named in pascal-case. Should this specific one be different? I mean what's the general naming rule for options and flags? How about `DEVMODE_DisplayFlags`?

Reply:
Our official naming rules are in our CONTRIBUTING.md doc. We add to that as undocumented cases become common. In the interim (like this case, which is the first like it that I've encountered) we muddle through, trying to invent something as consistent with precedent as possible.

We use `Flags` suffixes for flags enums, otherwise Options is common. We always name enums exactly as they are named in the header files, when they are actually defined, but they rarely are. Usually they are just a bunch of loosely bundled `#define` constants.
If we knew that some of these enums were used in other contexts, we'd study those other contexts and promote the enum to a top-level one that doesn't only serve this particular struct.

Reply:
So even this enum, has one value defined as `DM_INTERLACED` while the other is `DMDISPLAYFLAGS_TEXTMODE`...
I wasn't sure what your verdict was, are you fine with _DEVMODE_Display**Flags**_?